_extras/guide.md

```diff
@@ -89,18 +89,18 @@ Next,
 create the database that will be used:
 
 ~~~
-$ sqlite3 survey.sqlite .read bin/create-db.sql
+$ sqlite3 survey.db .read bin/create-db.sql
```

Comment:
We'll be moving to calling this file `survey.sqlite` everywhere soon (see #156) -- can you remove these changes?